src/pages/jiyeon/Login/LoginJiyeon.js

```diff
@@ -1,7 +1,61 @@
 import React from 'react';
+import './LoginJiyeon.scss';
+import { Link } from 'react-router-dom';
+import '../../../styles/jiyeon/mixin.scss';
 
-function LoginJiyeon() {
-  return <h1>hi</h1>;
-}
+import { useState } from 'react';
+
+const LoginJiyeon = () => {
+  // const [value, setValue] = useState();
+  const [id, setUserId] = useState('');
+  const [pw, setUserPw] = useState('');
+  const [loginBtnstate, setLoginBtnstate] = useState(false);
+
+  const saveUserId = e => {
+    setUserId(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    setUserPw(e.target.value);
+  };
+
+  const loginBtn = () => {
+    //if (id.indexOf(@) && pw.length > 4) =>true
+    return id.includes('@') && pw.length > 4 ? true : false;
```

Comment:
```suggestion
    return id.includes('@') && pw.length > 4
```
해당 조건의 반환값이 boolean 값이기 때문에 삼항연산자로 true 혹은 false를 return 하지 않아도 됩니다!